metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -662,8 +663,16 @@ public void apply(final Metafix metafix, final Record record, final List<String>
         public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
             record.transform(params.get(0), s -> s.toUpperCase());
         }
+    },
+    uri_encode {
+        @Override
+        public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
+            URL_ENCODER.setPlusForSpace(false);
+            record.transform(params.get(0), URL_ENCODER::process);
+        }
     };
 
+    private static final URLEncode URL_ENCODER = new URLEncode();
```

Comment:
There is no connection with `NAMED_GROUP_PATTERN`. Please move it to the end, separated by an empty line below `RANDOM`.

_If_ it should be a singleton at all, especially when we're making it configurable (see above).

Reply:
Good point. Moved instantiation into method in [c20c2a9](https://github.com/metafacture/metafacture-fix/pull/314/commits/c20c2a945d60fd3a52f3a218bc752b4561069a90).